### PR TITLE
Initial content for the Skupper operator bundle

### PR DESCRIPTION
**Pull request by fgiorgetti**

### CODE REVIEW

File: bundle/manifests/skupper-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+          - serviceaccounts
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        serviceAccountName: skupper-service-controller
```
Comment (grs) [inline bundle/manifests/skupper-operator.clusterserviceversion.yaml:195]: Should the skupper-service-controller and skupper-router serviceaccounts be specified here? The site controller will create them, and in the case we are installing the operator for all namespaces, the serviceaccounts here will not be applied to the namespaces in which skupper is activated anyway (only the site controller can do that).

File: bundle/manifests/skupper-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+  keywords:
+  - skupper
+  - service
+  - mesh
+  - van
+  links:
+  - name: Skupper Operator
+    url: https://github.com/skupperproject/skupper-operator
+  maintainers:
+  - email: customerservice@redhat.com
```
Comment (grs) [inline bundle/manifests/skupper-operator.clusterserviceversion.yaml:289]: This is probably not the right email for the upstream. Perhaps put in the skupper google group?

**Review by grs — APPROVED** [review]
Very nice!

File: bundle/manifests/skupper-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (fgiorgetti) [reply bundle/manifests/skupper-operator.clusterserviceversion.yaml:195]: Nope, I will remove them and test again before merging.

File: bundle/manifests/skupper-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (fgiorgetti) [reply bundle/manifests/skupper-operator.clusterserviceversion.yaml:289]: I wasn't sure but it makes sense to me as well. I will update it.

**Review by grs — APPROVED** [review]

**Pull request merged**